src/item_factory.cpp

```diff
@@ -1464,9 +1482,9 @@ void Item_factory::finalize()
             continue;
         }
         const itype_id &result = rec.result();
-        auto it = m_templates.find( result );
-        if( it != m_templates.end() ) {
-            it->second.recipes.push_back( p.first );
+        auto it = find_template_list( result );
```

Comment:
I would prefer if find_template_list did not use get_all_mod, and this one use case of it that needs to modify did.